style.css

```diff
@@ -10,12 +10,72 @@ body {
   overflow-x: hidden;
 }
 
+.nav {
+  z-index: 10;
+  position: fixed;
+  top: 10px;
+  right: 50px;
+}
+
+.hamburger.hide {
+  display: none;
+}
+
+.menu_mobile.active {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  gap: 20px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  padding: 100px;
+  backdrop-filter: blur(8px);
+  height: 100vh;
+  background-color: rgba(60, 58, 57, 0.9);
+  padding-top: 40px;
+  padding-left: 40px;
+}
+
+.menu_list {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  gap: 40px;
+  height: 100%;
+  padding: 0;
+}
+
+.menu_link {
+  border-bottom: 2px solid #f4f5f7;
+  display: block;
+  padding-bottom: 5px;
+  font-weight: 700;
+  font-size: 2rem;
+  color: #fff5e1;
+}
```

Comment:
- [ ] Great job implementing changes by the previous reviewer. However, there're few things you still need to work on.
- First of them, you're required to stick to the Figma design as close as possible. The color for the border-bottom you used is white as opposed to the color on the figma design. The hex code for the color used on the figma is this :point_right: `background: #6F6C6B;`. Kindly work on that.
- Secondly, the width of the border-bottom is extended down to the under `X` icon on the figma design as opposed to how it is on your design. Kindly look into that too and fix.

### Your Design
![your-menu](https://user-images.githubusercontent.com/26597432/232739715-dc761b62-41ca-41a2-a7fd-52cfb2c55194.png)

### Figma Design
![temp-menu](https://user-images.githubusercontent.com/26597432/232739743-72a92e45-8ca8-4372-a10b-8687b4c52766.png)